reference.md

```diff
@@ -5,4 +5,26 @@ permalink: /reference/
 
 ## Glossary
 
-FIXME
+The following list captures terms that need to be added to this glossay. This is a great way to contribute.
+
+- [ ] [Accelerator](https://en.wikipedia.org/wiki/Hardware_acceleration)
+- [ ] [Beowulf cluster](https://en.wikipedia.org/wiki/Beowulf_cluster)
+- [ ] [Central processing unit](https://en.wikipedia.org/wiki/Central_processing_unit)
+- [ ] [Cloud computing](https://en.wikipedia.org/wiki/Cloud_computing)
+- [ ] [Cluster](https://en.wikipedia.org/wiki/Computer_cluster)
+- [ ] [Distributed memory](https://en.wikipedia.org/wiki/Distributed_memory)
+- [ ] [Grid computing](https://en.wikipedia.org/wiki/Grid_computing)
+- [ ] [High availability computing](https://en.wikipedia.org/wiki/High_availability)
+- [ ] [High performance computing](https://en.wikipedia.org/w/index.php?title=High-performance_computing&redirect=no)
+- [ ] [Interconnect](https://en.wikipedia.org/wiki/Supercomputer_architecture)
+- [ ] [Node](https://en.wikipedia.org/wiki/Node_(computer_science))
+- [ ] [Parallel](https://en.wikipedia.org/wiki/Parallel_computing)
+- [ ] [Serial](https://en.wikipedia.org/wiki/Serial_computer)
+- [ ] [Server](https://en.wikipedia.org/wiki/Server_(computing))
+- [ ] [Shared memory](https://en.wikipedia.org/wiki/Shared_memory)
+- [ ] [Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager)
+- [ ] [Supercomputer](https://en.wikipedia.org/wiki/Supercomputer) -- "[a major scientific instrument](https://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.htmlhttps://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.html)"
```

Comment:
https://www.hpcnotes.com/2015/10/essential-analogies-for-hpc-advocate.html is (now?) a broken link :(

Reply:
Whoops, must have copy-pasted wrong, abort comment